config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
-  resources :faqs, only: :index
```

Comment:
For resources with only a single action, let's use `get '/foo', to: 'foo#index', as: 'foo'` instead of `resources`.